### PR TITLE
Update to 1.20.1 and add russian translation

### PR DESCRIPTION
**Pull request by Rozbiynk**

### CODE REVIEW

**Review by hf02 — CHANGES_REQUESTED** [review]
thx, just a single thing.

File: src/main/resources/fabric.mod.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
 	"schemaVersion": 1,
 	"id": "scroll-for-worldedit",
-	"version": "1.1.3",
+	"version": "${version}",
```
[the remark this answers is not in the document]
Comment (Rozbiynk) [reply src/main/resources/fabric.mod.json:4]: what wrong?

[the remark this answers is not in the document]
Comment (Rozbiynk) [reply src/main/resources/fabric.mod.json:4]: oh, I got it

**Comment by Rozbiynk** [comment]
is now all good?

**Comment by Rozbiynk** [comment]
oh

**Comment by hf02** [comment]
> is now all good?

yep, thanks!

**Pull request merged**